CONTRIBUTING_TR.md

```diff
@@ -0,0 +1,12 @@
+HaxeFlixel'e katkıda bulunduğunuz için teşekkürler!
+## Hata bildirme
+
+Hata bildirdiğiniz zaman, size sunulan [sorun şablonu](https://github.com/HaxeFlixel/flixel/blob/dev/.github/ISSUE_TEMPLATE.md)nu basit bir biçimde doldurun.
```

Comment:
```suggestion
Hata bildirdiğiniz zaman, size sunulan [sorun şablonu](https://github.com/HaxeFlixel/flixel/blob/dev/.github/ISSUE_TEMPLATE.md) nu basit bir biçimde doldurun.
```
all throughout the new files, I see spaces after links have been removed, this makes it look odd, like in the image below
<img width="622" alt="Screenshot 2023-06-02 at 4 21 45 PM" src="https://github.com/HaxeFlixel/flixel/assets/2609513/c31ea373-99da-4374-8a50-7835c0a0c204">